charts/handbrake/0.0.1/values.yaml

```diff
@@ -0,0 +1,87 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+  strategy:
+    type: Recreate
+
+  service:
+    port:
+      port: 5800            #Required for web GUI
+    additionalServices:
+      - enabled: true
+        nameSuffix: handbrake-vnc
+        type: ClusterIP
+        port:
+          port: 5900        #Optional, only for vnc clients.
+          name: handbrake-vnc
+          protocol: TCP
+          targetPort: 5900
+  
+
+  #All values here are set as the docker defaults.
+  env:
+  # Permissions Settings
+    USER_ID: "1000"
+    GROUP_ID: "1000"
+    SUP_GROUP_IDS:
+    UMASK:    
+  # General Settings
+    #TZ: Etc/UTC    #Do we need this?
+    KEEP_APP_RUNNING: "0"
+    APP_NICENESS: 
+    CLEAN_TMP_DIR: "1"
+  # GUI Settings
+    DISPLAY_WIDTH: "1280"
+    DISPLAY_HEIGHT: "768"
+    SECURE_CONNECTION: "0"
+    VNC_PASSWORD:
+    X11VNC_EXTRA_OPTS:
+    ENABLE_CJK_FONT: "0"
+  # Automated Conversion Preset
+    AUTOMATED_CONVERSION_PRESET: General/Very Fast 1080p30
+    AUTOMATED_CONVERSION_FORMAT: mp4
+    AUTOMATED_CONVERSION_KEEP_SOURCE: "1"
+    AUTOMATED_CONVERSION_VIDEO_FILE_EXTENSIONS:
+    AUTOMATED_CONVERSION_NON_VIDEO_FILE_ACTION: ignore
+    AUTOMATED_CONVERSION_NON_VIDEO_FILE_EXTENSIONS: jpg jpeg bmp png gif txt nfo
+    AUTOMATED_CONVERSION_OUTPUT_DIR: /output
+    AUTOMATED_CONVERSION_OUTPUT_SUBDIR:
+    AUTOMATED_CONVERSION_OVERWRITE_OUTPUT: "0"
+    AUTOMATED_CONVERSION_SOURCE_STABLE_TIME: "5"
+    AUTOMATED_CONVERSION_SOURCE_MIN_DURATION: "10"
+    AUTOMATED_CONVERSION_CHECK_INTERVAL: "5"
+    AUTOMATED_CONVERSION_MAX_WATCH_FOLDERS: "5"   ##If we let users set infinite watch folders, we need to update this dynamically or expose to UI to let them update it.
+    AUTOMATED_CONVERSION_NO_GUI_PROGRESS: "0"
+    AUTOMATED_CONVERSION_HANDBRAKE_CUSTOM_ARGS:
+    AUTOMATED_CONVERSION_INSTALL_PKGS:
```

Comment:
I know you might want some of these exposed. 
The way you did it with collabora works fine and you can add both the option to set custom (unlimited) environment variables (example is in multiple apps) AND set env vars the way you did in collabora (thats the reason both ways of doing env vars are supported at the same time.

Reply:
I'll should probably slim it down, to more "general" use and leave the rest to the user if he want to add from the "unlimited" option.

Reply:
Yes that would be prefered indeed... to not bloat the install interface :)